app/views/layouts/application.html.erb

```diff
@@ -22,6 +22,26 @@
     <%= render 'layouts/footer' %>
 
     <script>
+      $("input#search-input").on({
+        keydown: function(e) {
+          if (e.which === 32)
```

Comment:
the `which` attributes has been deprecated in favor of the `key` attribute. It's now compatible across all browsers ([see here](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility)) so lets start using it? I think the replacement would be `e.key === " "` but please double check!

Reply:
> @avsingh999 I requested some changes! Let me know your thoughts.
> 
> Also can you please open a new issue for this and reference it instead of the current one you're referencing. It makes it a lot easier to review and people are less likely to duplicate issues in this case

@sashadev-sky  Sorry I had added wrong issue number. #520 is correct.
thank you